homeassistant/components/netgear_lte/sensor_types.py

```diff
@@ -6,6 +6,19 @@
 SENSOR_UNITS = {
     SENSOR_SMS: 'unread',
     SENSOR_USAGE: 'MiB',
+    'radio_quality': '%',
+    'rx_level': 'dBm',
+    'tx_level': 'dBm',
+    'upstream': None,
+    'wire_connected': None,
```

Comment:
Actually, one second thought shouldn't `wire_connected` and `mobile_connected` be binary sensors?

Reply:
Good point. For `mobile_connected`, states disconnected/standby/connected have been observed. Maybe I should name it `mobile_status` instead?

The `wire_connected` is currently binary but I am guessing we could discover more states in the future (from the undocumented API). I am not sure what to do in such a case but I would probably rename it along with the first one.

It seems like `roaming` would indeed be binary, though.

I will think a bit about this and make a follow-up PR before the next beta. Thanks!